mobile/App.js

```diff
@@ -14,6 +16,7 @@ export default function App() {
         <Tab.Screen name="Map" component={MapScreen} />
         <Tab.Screen name="Gallery" component={GalleryScreen} />
         <Tab.Screen name="Submission" component={SubmissionScreen} />
+        <Tab.Screen name="Zines" component={ZinesScreen} />
       </Tab.Navigator>
     </NavigationContainer>
   );
```

Comment:
For some reason, you don't have the stack navigator here? Might have been an error from a merge commit